app/assets/stylesheets/partials/_menus.scss

```diff
@@ -1,9 +1,69 @@
-.dropdown-menu {
-  .menu {
-    i {
-      float: right;
-      margin-top: 4px;
+.menu-large {
+  position: static !important;
+}
+
+.megamenu {
+  padding: 10px 0;
+
+  &.fw {
```

Comment:
I think `full-width`is more descriptive, but I guess this is only used in one partial view.